package.json

```diff
@@ -59,7 +59,7 @@
         "glob": "7.1.3",
         "gulp-filter": "5.1.0",
         "insight": "0.10.1",
-        "jhipster-core": "4.3.0",
+        "jhipster-core": "4.4.0",
```

Comment:
5.0.0 might be better here (same content)

Reply:
What about the breaking changes? Don't we have to update anything in the generator?

Reply:
Nop! this is the good kind of breaking change, doesn't impact the generator.

Reply:
great, what about the custom annotation stuff, I saw a comment somewhere discussing aout changing the entity json files to have options inside an `options` key? was that done?

Reply:
Also for the `SEARCH_ENGINE` change doesn't it break existing JDL files? if it has `searchEngine` in it? given that we will update and release this in a minor JHipster version may be we need to add backward compatibility for JDL files that still uses `searchEngine`. WDYT?

Reply:
@deepu105 about the first question, I didn't see any comment so I can't relate, nor have I seen work about this `options` key.
About the second one: in the JDL, you still use the `search` keyword. Work has been done to harmonize what's written in a JDL file and the internal name of the option holding the value. The breaking change is about what's exposed by BinaryOptions.

Reply:
To be honest, the first point bothers me. @yelhouti do you have any input about this? I don't remember about such a key :(

Reply:
@MathieuAA : the `options` key have been suggested by you in [this comment](https://github.com/jhipster/jhipster-core/issues/320#issuecomment-485381566), but this has not been done (yet?).

Reply:
@murdos ahah! yep. Now I remember. That was a suggestion for using custom annotations, but doesn't break existing options

Reply:
perfect

Reply:
@MathieuAA @deepu105 , options field is only used for custom annotations for backward compatility reasons, once we switch to JDL centric approach, I could go through the code again to harmonize the logique.

Reply:
@yelhouti nice :)